tests/integration-tests/configs/new_os.yaml

```diff
@@ -72,6 +77,7 @@ test-suites:
     test_createami.py::test_build_image:
       dimensions:
         - regions: ["eu-west-3"]
+          schedulers: [ "slurm" ]
```

Comment:
Why is this change required now?

Reply:
This change is optional

Reply:
Minor: This change is something that we missed when adding this new_os.yaml, if this is not added the test fails with missing schedulers dimensions